resources/views/api/includes/contents/newapp.blade.php

```diff
@@ -0,0 +1,61 @@
+@if ( Auth::check() )
+
+        <!-- top nav -->
+    @include("api.includes.sections.top_nav")
+
+    <div class='row'>
+    	<div class="col s8">
+        	<div id="main-new-one">
+	    		<div class='form-feature'>
+			    	<form>
+
+			    		<h5>Application name</h5>
+
+				    		<div class='form-group'>
```

Comment:
**Code Quality - Nitpick**
@andela-araimi Checkout Laravel Form helpers (Input::old), for better user experinece.